projects/step-core/src/lib/client/augmented/services/augmented-keywords-service.ts

```diff
@@ -21,7 +22,36 @@ export class AugmentedKeywordsService extends KeywordsService {
     );
   }
 
-  constructor(override httpRequest: BaseHttpRequest, private _tableRest: TableApiWrapperService) {
-    super(httpRequest);
+  override newFunctionTypeConf(type: string): Observable<Function>;
+  override newFunctionTypeConf(type: string, serviceRoot: string): Observable<Function>;
+  override newFunctionTypeConf(type: string, serviceRoot?: string): Observable<Function> {
+    return this.httpRequest.request({
+      method: 'GET',
+      url: `/${serviceRoot}/types/${type}`,
+    });
+  }
```

Comment:
@neogucky Something wrong with this approach, that was originally implemented.
We shouldn't change that path the root path of the api services.
Previously it was done for `masksList` in pdf-test plugin, but EE has it's own endpoint to invoke these methods.
So in case of plugin another service should be used, instead of changing path in the keyword service

@neogucky, @bojidaryovchev 
Is it possible to rollback the `function-dialogs.service` to direct `httpClient` usage, and then fix this problem correctly under the separate task?

Reply:
This is approach is based on the original approach used in the AJS version. I agree that calling the correct endpoint without a wild card would be preferrable, also we should limit augmenting things whenever we can because this undermines the Swagger concept.

I don't get the point regarding EE, the service is in the core so should work fine with both versions. Can you elaborate @dvladir ?